examples/bokeh/bokeh_plot_bpv_tstat.py

```diff
@@ -0,0 +1,10 @@
+"""
+Bayesian p-value Posterior plot
```

Comment:
Should this have a different name from the other, to indicate it is using a T stat?